include/exadg/aero_acoustic/user_interface/parameters.h

```diff
@@ -0,0 +1,105 @@
+/*  ______________________________________________________________________
+ *
+ *  ExaDG - High-Order Discontinuous Galerkin for the Exa-Scale
+ *
+ *  Copyright (C) 2023 by the ExaDG authors
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *  ______________________________________________________________________
+ */
+
+#ifndef EXADG_AERO_ACOUSTIC_USER_INTERFACE_INPUT_PARAMETERS_H_
+#define EXADG_AERO_ACOUSTIC_USER_INTERFACE_INPUT_PARAMETERS_H_
+
+// deal.II
+#include <deal.II/base/conditional_ostream.h>
+#include <deal.II/base/parameter_handler.h>
+
+namespace ExaDG
+{
+namespace AeroAcoustic
+{
+enum class FluidToAcousticCouplingStrategy
+{
+  Undefined,
+  NonNestedMGRestriction
```

Comment:
I do not like the `MG` naming here. We might have this naming in dealii, but here we should speak to the ExaDG user and not to the dealii expert in my opinion. Why not something simple like `NonNestedGridTransfer`?

Reply:
In the aero-acoustic community, this transfer is called conservative interpolation. Would this name also be OK for you?

Reply:
yes

Reply:
[f6b4340](https://github.com/exadg/exadg/pull/635/commits/f6b43402d57c58301f9346d1adfdcf96e0f8fd91)